pandas/core/frame.py

```diff
@@ -531,6 +531,7 @@ class DataFrame(NDFrame, OpsMixin):
         will perform column selection instead.
     dtype : dtype, default None
         Data type to force. Only a single dtype is allowed. If None, infer.
+        If `data` is DataFrame then is ignored.
```

Comment:
```suggestion
        If ``data`` is DataFrame then is ignored.
```

Reply:
Why do we need double backticks if elsewhere they are single?